src/components/admonition/admonition.scss

```diff
@@ -0,0 +1,41 @@
+.admonition {
+    display: flex;
+    width: 100%;
+
+    &.danger {
+        background-color: rgb(var(--color-red-dark));
+    }
+    &.info {
+        background-color: rgb(var(--color-cyan-default));
+    }
+    &.note {
+        background-color: rgb(var(--color-gray-default));
+    }
+    &.tip {
+        background-color: rgb(var(--color-green-dark));
+    }
+    &.warning {
+        background-color: rgb(var(--color-orange-dark));
+    }
+
+    .admonition-side {
+        mix-blend-mode: screen;
+        background-color: rgb(var(--contrast-800));
+        padding: 0.25rem;
+    }
+    .admonition-main {
+        mix-blend-mode: screen;
+        background-color: rgb(var(--contrast-300));
+        padding-left: 0.5rem;
+        width: 100%;
+
+        .admonition-main-title {
+            margin: 0.1rem;
+            margin-top: 0.25rem;
+        }
+        .admonition-main-description {
+            margin: 0.1rem;
```

Comment:
This is a matter for @Kiarokh and @LucyChyzhova, but is it really meaningful to have a 1.6px margin?